packages/glimmer-compiler/lib/template-compiler.ts

```diff
@@ -10,6 +10,10 @@ export interface CompileOptions {
   moduleName?: string;
 }
 
+function isTrustingValue(value) {
```

Comment:
@krisselden exactly. That's why I used this naming. 

<table>
<tr>
  <th></th><th>Trusted Value</th><th>Untrusted Value</th>
</tr>
<tr>
  <th>Trusting Operation (triple curlies)</th><td>✔️</td><td>✔️</td>
</tr>
<tr>
  <th>Cautious Operation (double curlies)</th><td>✔️</td><td>✖️</td>
</tr>
</table>

Trusted Values:

* Literals in a template
* Escaped strings using `escapeExpression()`
* Strings marked as safe using `htmlSafe()`

Untrusted Values:

* JavaScript strings that were neither escaped nor marked as safe